### PR TITLE
User not available

### PR DESCRIPTION
**Pull request by GellertPalfi**

### CODE REVIEW

File: action_server/time_table.py [hunk 1]
```diff
@@ -147,11 +157,25 @@ def discard_currently_discussed_bottom_range(self):
                 # this shouldn't really happen...
                 pass
             else:
-                self.current_dtrl.ladder = [new_prev[0]] + self.current_dtrl.ladder[1:]
+                self.current_dtrlbot_free.ladder = [new_prev[0]] + self.current_dtrl.ladder[1:]
```
Comment (nsomabalint) [inline action_server/time_table.py:160]: Where is this memeber variable defined?

**Comment by nsomabalint** [comment]
```
Your input ->  időpontot szeretnék                                                                                                                                                                                                       
Okés. Mikor lenne jó?
Esetleg holnap 12 óra és délután 6 óra között jó valamikor?
Your input ->  sajnos holnap nem jó                                                                                                                                                                                                      
Rendben, holnap 12 óra és délután 6 óra között működhet. Mikor lenne jó pontosan?
```
Intents are correctly detected here, but it looks to me that the `recommend_other_date`  action recommends dates which the user had already refused.

**Comment by GellertPalfi** [comment]
Unfortunately, because of the small training sample size, this does happen, when you mention dates or time intervals the bot thinks you are trying to book an appointment.
 
image

for more detailed information, please see my slack comment about this matter

File: action_server/time_table.py (hunk 1 above)
Comment (GellertPalfi) [reply action_server/time_table.py]: it was an accidental typo, now fixed

**Pull request merged**